### DISCUSSION TITLE
Figma kit for Skeleton v1.0 🚀 [Unofficial]

### DISCUSSION THREAD

**Discussion by mmailaender** [skeletonlabs/skeleton#2162] [Show and tell]
Finally, the [Figma kit v1.0](https://www.figma.com/community/file/1266774078142216791) landed, and the best part is that it is free for everyone.

If the Figma Kit helps you, please consider a [Skeleton sponsorship](https://github.com/sponsors/skeletonlabs). It allows the Skeleton Team to provide you with long-term amazing features while making it free and available for everyone.

# Features
- Component variants
- Theme support with variables
- Composable with slots
- Tutorial (Only Alerts at the moment)
- Playground

# Components
## Tailwind
- Core, Typography
- Forms: Input, Text Area, Select, Checkbox, Radio, Material Style Input, Input Group, Range
- Alert, Badges, Breadcrumbs, Button, Button Group, Cards, Chips, Lists, Logo Clouds, Placeholder
- Chats, Gradient headings, Image Layouts, Scroll Containers

## Svelte
- Accordions, App Bar, App Rail, App Shell
- Autocomplete, Avatars, File Buttons, File Dropzone, Input Chips, Listboxes, Paginators
- Progress Bars, Progress Radials, Ratings, Radio Groups, Range Sliders
- Slide toggles, Steppers, Tabs, Tables, Tree Views

## Utilities
- Code Blocks, Drawers, Lightswitch, Modals, Popups, Toasts, Table of Contents

**Comment by bertramdesigns** [comment]
Awesome! Love it! 

Found a few bugs in the system. How would you like us to handle reports for the Figma kit?

**Reply by mmailaender** [reply]
@bertramdesigns feel free to share it directly here 😊

**Reply by bertramdesigns** [reply]
Sorry, went on holiday and work went on pause. 

I have invited you to the project where I report all of the bugs I have found :) 

For now:
<img width="338" alt="Screenshot 2024-01-11 at 10 42 04" src="https://github.com/skeletonlabs/skeleton/assets/60759558/e13660d3-9731-4927-ac8f-010475d7a416">

**Reply by mmailaender** [reply]
@bertramdesigns your mentioned points should be fixed with the latest version. 

Btw. I haven't received any notification or link to your mentioned project so was not able to check it out.

**Reply by bertramdesigns** [reply]
Wonderful to hear :) 
I am not sure why the notifications are not coming in. Try accessing it [here](https://github.com/users/bertramdesigns/projects/2)

I have only added a couple more items. My focus has been on rebuilding the tree view component. Unfortunately, it is missing some of the robustness and flexibility of systems like [Ant](https://ant.design/components/tree#components-tree-demo-line).

**Reply by mmailaender** [reply]
There is also an issue addressing some of the current limitations of the tree view: https://github.com/skeletonlabs/skeleton/issues/2358
If you're considering rebuilding the tree view component, then maybe you want to participate in that issue. 🙂 @bertramdesigns

**Reply by bertramdesigns** [reply]
Will do. Getting through the "figmatism" now, but I will join the issue once I figure out an improved designer experience with the component and do a round on usability.

WIP is [here](https://www.figma.com/file/Oq4s5980EsqeBEsVEVjz25/Skeleton-(Community)?type=design&node-id=526%3A358942&mode=design&t=xoIvh3B01rjIG00v-1)
